Original patch from Gentoo:
gentoo/src/patchsets/binutils/2.19/76_all_use-new-ld-dtags.patch

diff -durN binutils-2.19.orig/ld/ldmain.c binutils-2.19/ld/ldmain.c
--- binutils-2.19.orig/ld/ldmain.c	2008-11-23 16:54:42.000000000 +0100
+++ binutils-2.19/ld/ldmain.c	2008-11-23 16:56:16.000000000 +0100
@@ -262,6 +262,7 @@
 
   link_info.allow_undefined_version = TRUE;
   link_info.keep_memory = TRUE;
+  link_info.new_dtags = TRUE;
   link_info.combreloc = TRUE;
   link_info.strip_discarded = TRUE;
   link_info.emit_hash = TRUE;
